src/main/java/seedu/address/logic/commands/facil/FacilCommand.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.commands.facil;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.facilitator.Facilitator;
+import seedu.address.model.facilitator.NameContainsKeywordsPredicate;
+
+import static java.util.Objects.requireNonNull;
+
+public abstract class FacilCommand extends Command {
+    public static final String ADD_COMMAND_WORD = "add";
+    public static final String EDIT_COMMAND_WORD = "edit";
+    public static final String LIST_COMMAND_WORD = "list";
+    public static final String FIND_COMMAND_WORD = "find";
+    public static final String DELETE_COMMAND_WORD = "delete";
+
```

Comment:
Actually I think that maybe can put these in Command class since other commands such as ModCommand will also have  some of these command word.